spring/boot-autoconfigure/build.gradle

```diff
@@ -11,6 +11,35 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter'
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
 
+    testCompile project(':grpc')
     testCompile 'org.springframework.boot:spring-boot-starter-actuator'
     testCompile 'org.springframework.boot:spring-boot-starter-test'
 }
+
+protobuf {
```

Comment:
I thought we do not need this at all because `gradle-scripts` takes care of this already. Does the build fail without this section?